.github/.github_changelog_generator

```diff
@@ -1,13 +1,15 @@
 issues=false
 pr-wo-labels=false
 author=false
-breaking-label=Changed
+breaking-label=### Changed
 breaking-labels=Breaking change
-enhancement-label=Added
+enhancement-label=### Added
 enhancement-labels=Added
-bugs-label=Patch
+bugs-label=### Patch
 bug-labels=Patch
+deprecated-prefix=### Deprecated
 deprecated-labels=Deprecated
+removed-prefix=### Removed
```

Comment:
Why are they not consistent with their options? Here `-prefix` is used instead of `-label`, is that correct?